scripts/setup-k8s-master.sh.in

```diff
@@ -72,9 +72,6 @@ export KUBECONFIG=/etc/kubernetes/admin.conf
 # Grant the "admin" user complete access to the cluster
 kubectl create clusterrolebinding admin-cluster-binding --clusterrole=cluster-admin --user=admin
 
-# Label this node as a master
-kubectl label node "${HOSTNAME}" "kubernetes.io/role=master"
```

Comment:
this happens as part of kubeadm?

Reply:
kubeadm sets 'node-role.kubernetes.io/master=' which from what I can tell is the prescribed node labeling going forward, since it allows for a node to have multiple roles. 'kubernetes.io/role=master' appears to be deprecated.